lib/ae_mdw/database.ex

```diff
@@ -162,8 +212,17 @@ defmodule AeMdw.Database do
   end
 
   @spec write(table(), record()) :: :ok
+  def write(tab, record) when use_rocksdb?(tab) do
+    :ok = RocksDbCF.put(tab, record)
```

Comment:
No need to pattern-match the result here

Reply:
yes, some of the returns were already checked by `RocksDbCF` but it was a good thing we review it.